wpengine/faustjs
# [Merl 1149] Add TextControl field

## Tasks



## Description

* `registerFaustBlock` Edit function now handles `text` control fields.

## Related Issue(s):

https://github.com/wpengine/faustjs/issues/1522

## Testing

Follow testing instruction from this PR:

https://github.com/wpengine/faustjs/pull/1559

* Make sure to add a field with `type:  text`.
* Create a new block. You should be able to see the text fields for that attribute. You should be able to update the field and the field should persist when saving the post/page.


## Screenshots



<img width="1062" alt="Screenshot 2023-09-05 at 14 39 32" src="https://github.com/wpengine/faustjs/assets/328805/4b3bd239-33bf-4b67-a7b9-82ac4e054d69">

<img width="1023" alt="Screenshot 2023-09-05 at 14 40 07" src="https://github.com/wpengine/faustjs/assets/328805/0d9a2e8a-f5cd-4f3f-a59f-85e8383eda7a">

## Documentation Changes



## Dependant PRs
